### ISSUE TITLE
WP SC admin UI all munged up

### ISSUE THREAD

**Issue by gws3** [Automattic/jetpack#34066]
### Impacted plugin

Super Cache

### Quick summary

Odd, no changes made, the WP SC admin page is not right, UI is nonfunctional. I know WP auto updated recently, any hints on how to fix?  Looks like css error somewhere.
![wpf](https://github.com/Automattic/jetpack/assets/21205515/eee24544-82ed-4c2a-bedf-81beae3c52ee)

### Steps to reproduce

Open SC settings in wp-admin

### Impact

One

### Available workarounds?

No but the platform is still usable

**Comment by jeherve** [comment]
Do you use other plugins that may have their own custom admin page? Looking at your screenshot, it would appear as if the styles of another plugin were loaded in the WP Super Cache page as well.

Could you try to deactivate all other plugins, one at a time, and let us know if you find the culprit?

Thank you!

**Comment by gws3** [comment]
Hi J, as it ran clean yesterday and I didn't do any other updates, I'm going to have to guess for ya (I have a deadline and was forced to go to a backup for a demo this morning)...

Anyway, the only other issues I had at the time was odd behavior by Content Control plugin, https://wordpress.org/plugins/content-control/, and it does have a custom admin page. 

I also ran into a weird unable to save issue in child's function.php, made no sense, code was clean. 

Hope that helps. Not sure if I will reinstall SuperCache, there are a lot of little custom things that happen depending on user roles on this build, might not be that easy to stay on top of them.

Thanks

**Comment by jeherve** [comment]
That doesn't seem to be Content Control, I tried running both WP Super Cache and that plugin and saw no issues:

<img width="1407" alt="image" src="https://github.com/Automattic/jetpack/assets/426388/a36e44d8-0203-4898-ab08-ddeb755e8faa">

If it's not a conflict between plugin pages, I wonder if something else could be at play here. Could you check for errors on the page, as explained here:
https://wordpress.org/support/article/using-your-browser-to-diagnose-javascript-errors/

I'm hoping maybe there'll be errors there that can help us. Let me know what you find!

**Comment by gws3** [comment]
I tried to replicate it but have failed, I thought something changed overnight I was not aware of, besides the Wordpress update. However in the past it has usually ended up being that I changed something and forgot about it or hadn't noticed another error prior to the one I get stuck on. I think now maybe the Content Control was not functioning correctly after I installed WP Super Cache,, and I just didn't notice it, as there are a lot of ways we have it masking certain assets. Wish I could be more help, but I do thank you for your help.
